### DISCUSSION TITLE
http verification fails with http_image.jpg

### DISCUSSION THREAD

**Discussion by mananas** [ultralytics/yolov5#8990] [Q&A]
URL verification currently in
im, f = Image.open(requests.get(im, stream=True).raw if str(im).startswith('http') else im), im
which fails with e.g. http_image.jpg, since the pure "http" in the beginning of the image name launches http request.
Maybe the verification should be str(im).startswith('http:') ?

**Comment by glenn-jocher** [comment]
@mananas we can update this to `.startswith(('http://', 'https://'))`. Please submit a PR, thank you!
